backend/appsettings.Development.json

```diff
@@ -4,5 +4,8 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "ConnectionStrings": {
+    "VibesDb": "Server=localhost;Database=AzureVibesDb;User Id=sa;Password=yourStrong(!)Password;;TrustServerCertificate=true"
```

Comment:
@jonasbjoralt What was needed here again for dev-mode?

Reply:
This is fine I think